js/sales-app.js

```diff
@@ -19,32 +28,35 @@ function makeHeaderRow() {
 
 function StoreSales(location, minCustomers, maxCustomers, avgCookiesPerCustomer) {
   this.location = location;
-  this.minCustomers = minCustomers;
-  this.maxCustomers = maxCustomers;
-  this.avgCookiesPerCustomer = avgCookiesPerCustomer;
-
+  this.minCustomers = parseInt(minCustomers);
+  this.maxCustomers = parseInt(maxCustomers);
+  this.avgCookiesPerCustomer = parseInt(avgCookiesPerCustomer);
```

Comment:
The average number of cookies can be a float, so you should use `parseFloat` instead of `parseInt`.